common/pulp/common/error_codes.py

```diff
@@ -113,6 +113,9 @@
 PLP0043 = Error("PLP0043", _("Database 'write_concern' config can only be 'majority' or 'all'. "
                              "Refer to /etc/pulp/server.conf for proper use."), [])
 PLP0044 = Error("PLP0033", _("The target importer does not support the types from the source"), [])
+PLP0045 = Error("PLP0045", _("The importer %(importer_id)s indicated a failed response when "
+                             "uploading %(unit_type)s unit to repository %(repository_id)s."),
+                ['importer_id', 'unit_type', 'repository_id'])
```

Comment:
The line above this seems to be strange - the error is 0044 but the text is 0033. Furthermore, there does not seem to be an 0033 above. I wonder if this means that the one above this was really supposed to be 0033, which would mean your new one should be 0044. What do you think?